### DISCUSSION TITLE
Method ClientTwinProperties.GetLastUpdatedOnUtc

### DISCUSSION THREAD

**Discussion by chandra-rayaprol** [Azure/azure-iot-sdk-csharp#3155] [Q&A] [answered]
The method 'ClientTwinProperties.GetLastUpdatedOnUtc()' does not exist in the most recent 2.0.0-preview003 version.
Is there another way to get this property's value?

**Comment by drwill-ms** [comment, marked as answer]
I think what you are looking for is found under `ClientTwinProperties.Metadata.LastUpdatedOnUtc`.

**Reply by drwill-ms** [reply]
It looks like our migration guide is out of date on this. I'll fix it up.

**Comment by abhipsaMisra** [comment]
Thank you for trying out our `previews/v2` client.

We've now provided dedicated twin property metadata accessors: https://github.com/Azure/azure-iot-sdk-csharp/blob/f7bec7d78535dff52200a8f581e2710968e34e4a/iothub/service/src/Twin/Models/ClientTwinMetadata.cs#L20

Sample usage:
```c#
ClientTwin clientTwin = await s_serviceClient.Twins.GetAsync(deviceId).ConfigureAwait(false);
DateTimeOffset? lastUpdatedOnUtc = clientTwin.Properties.Reported.Metadata.LastUpdatedOnUtc;
```

**Comment by chandra-rayaprol** [comment]
Thank you for your prompt response. It is a welcome change.
